### PR TITLE
Bug 2098234: fix broken update server link

### PR DESCRIPTION
**Pull request by rhamilto**
And move all doc links to a single file for purposes of auditing so that we can more easily identify broken links in the future (yes, this is my idea of a fun thing to do for HACk week).  :)

### CODE REVIEW

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/cluster-setup-getting-started-card.tsx [hunk 1]
```diff
@@ -26,9 +27,9 @@ export const ClusterSetupGettingStartedCard: React.FC = () => {
     return null;
   }
 
-  const moreLinkURL = window.SERVER_FLAGS.documentationBaseURL
-    ? `${window.SERVER_FLAGS.documentationBaseURL}html/post-installation_configuration/post-install-machine-configuration-tasks`
-    : 'https://docs.okd.io/latest/post_installation_configuration/machine-configuration-tasks.html';
+  const moreLinkURL = getDocumentationURL(
```
Comment (jhadvig) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/cluster-setup-getting-started-card.tsx:30]: was this broken by the linter? If not lets make it into a one=liner

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/__tests__/cluster-setup-getting-started-card.spec.tsx [hunk 2]
```diff
@@ -71,11 +71,9 @@ describe('ClusterSetupGettingStartedCard', () => {
         href: '/monitoring/alertmanagerconfig',
       },
     ]);
-    expect(wrapper.find(GettingStartedCard).props().moreLink).toEqual({
+    expect(wrapper.find(GettingStartedCard).props().moreLink).toMatchObject({
```
Comment (jhadvig) [inline frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/__tests__/cluster-setup-getting-started-card.spec.tsx:74]: 👍

File: frontend/public/components/utils/documentation.tsx [hunk 3]
```diff
@@ -1,15 +1,98 @@
...
+export const DOC_URL_STORAGE_CLASSES_GLUSTERFS = `${KUBE_DOCS}/concepts/storage/storage-classes/#glusterfs`;
+export const DOC_URL_STORAGE_CLASSES_LOCAL = `${KUBE_DOCS}/concepts/storage/storage-classes/#local`;
+export const DOC_URL_STORAGE_CLASSES_OPENSTACK_CINDER = `${KUBE_DOCS}/concepts/storage/storage-classes/#openstack-cinder`;
+export const DOC_URL_STORAGE_CLASSES_PORTWORX_VOLUME = `${KUBE_DOCS}/concepts/storage/storage-classes/#portworx-volume`;
+export const DOC_URL_STORAGE_CLASSES_QUOBYTE = `${KUBE_DOCS}/concepts/storage/storage-classes/#quobyte`;
+export const DOC_URL_STORAGE_CLASSES_SCALEIO = `${KUBE_DOCS}/concepts/storage/storage-classes/#scaleio`;
+export const DOC_URL_STORAGE_CLASSES_STORAGEOS = `${KUBE_DOCS}/concepts/storage/storage-classes/#storageos`;
+export const DOC_URL_STORAGE_CLASSES_VSPHERE = `${KUBE_DOCS}/concepts/storage/storage-classes/#vsphere`;
+
+export const documentationURLs = {
```
Comment (jhadvig) [inline frontend/public/components/utils/documentation.tsx:26]: nit: maybe type the variable as well?
```suggestion
type documentationURLsTypes = {
  [key: string]: docURLs
}

export const documentationURLs: documentationURLsTypes = {
```

**Review by jhadvig — COMMENTED** [review]
@rhamilto the change looks great! Really nice HAC week project 👍 
Could be also incorporate [whats new link](https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/explore-admin-features-getting-started-card.tsx#L34)

File: frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/cluster-setup-getting-started-card.tsx (hunk 1 above)
Comment (rhamilto) [reply frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/cluster-setup-getting-started-card.tsx:30]: Linter insists on multi-line.

**Comment by rhamilto** [comment]
Thanks for the feedback, @jhadvig.  [Incorporated](https://github.com/openshift/console/compare/c6dc6705c10ca881a5a8a9029545814966ad7597..5c1beff4e2a8e98bb867825f5143e5f5adf5bf90).

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**